src/modules/addTodo.js

```diff
@@ -0,0 +1,27 @@
+import TodoList from './todoList.js';
+import getTodo from './getTodo.js';
+import setTodo from './setTodo.js';
+import renderAddedList from './renderAddedList.js';
+
+const addTodoBtn = () => {
+  const input = document.querySelector('.save-todo');
+  input.addEventListener('keypress', (event) => {
+    if (event.key === 'Enter') {
+      event.preventDefault();
+      const description = document.querySelector('.save-todo').value;
+      const completed = false;
+      const list = getTodo();
+      const index = list.todosize + 1;
+      if (description !== '') {
+        const todoList = new TodoList(description, completed, index);
+        setTodo(todoList);
+        renderAddedList(todoList);
+        // window.location.reload();
```

Comment:
You made a great job so far , but you committed old pieces of code as inline comments. They will make your project look messy. Kindly improve that by deleting these line 👌